react-components/src/hooks/useGroundPlaneFromScene.tsx

```diff
@@ -77,13 +77,24 @@ export const useGroundPlaneFromScene = (sceneExternalId: string, sceneSpaceId: s
     }
     const groundMeshes: CustomObject[] = [];
 
+    // New scale format was introduced as a bug in the Scenebuilder
+    // on August 1st, 2024. This is a temporary fix to handle both formats.
+    // This should be removed when we upgrade the Scene Data Model to a new version
+    const groundPlaneUsesNewScaleFormat = groundPlaneUseNewScaleFormat(
+      scene.sceneConfiguration.updatedAt
+    );
+
     scene.groundPlanes.forEach((groundPlane, index) => {
       if (groundPlaneTextures?.[index] === undefined) {
         return;
       }
       const texture = groundPlaneTextures[index];
       const material = new MeshBasicMaterial({ map: texture, side: DoubleSide });
-      const geometry = new PlaneGeometry(10000 * groundPlane.scaleX, 10000 * groundPlane.scaleY);
+
+      const scaleX = groundPlane.scaleX;
+      const scaleY = groundPlaneUsesNewScaleFormat ? groundPlane.scaleZ : groundPlane.scaleY;
+
+      const geometry = new PlaneGeometry(10000 * scaleX, 10000 * scaleY);
```

Comment:
Still a bit weird to have this random scaling factor here :sweat_smile: Should it be in a variable?

Reply:
Agreed, fixed in [60db811](https://github.com/cognitedata/reveal/pull/4743/commits/60db811605eb91e774af44c191d6ef66cd0ecedd)